src/openrct2/entity/Yaw.hpp

```diff
@@ -13,10 +13,21 @@
 
 namespace OpenRCT2::Entity::Yaw
 {
+    enum class SpritePrecision : uint8_t
+    {
+        Sprites0 = 0,
```

Comment:
As you are using this as a none state might as well call it such.

Reply:
Would you call it SpritesNone or NoneSprites

Reply:
Just call it SpritePrecision::None